README.md

```diff
@@ -191,13 +191,13 @@ $ brew install git-xargs
    every repo in your GitHub organization. Follow the same pattern to start running your own scripts and commands
    against your own repos!
 
-      ```bash
-      git-xargs \
-        --branch-name "test-branch" \
-        --commit-message "Testing git-xargs" \
-        --github-org <enter-your-github-org-name> \
-        touch git-xargs-is-awesome.txt
-      ```
+   ```bash
+   git-xargs \
+     --branch-name "test-branch" \
+     --commit-message "Testing git-xargs" \
+     --github-org <enter-your-github-org-name> \
+     touch git-xargs-is-awesome.txt
+   ```
```

Comment:
```suggestion
      ```bash
      git-xargs \
        --branch-name "test-branch" \
        --commit-message "Testing git-xargs" \
        --github-org <enter-your-github-org-name> \
        touch git-xargs-is-awesome.txt
      ```
```